src/nameserver/system_table_test.cc

```diff
@@ -60,9 +60,14 @@ TEST_F(SystemTableTest, SystemTable) {
     ns_client.Init();
     std::vector<::openmldb::nameserver::TableInfo> tables;
     std::string msg;
-    ASSERT_TRUE(ns_client.ShowTable("", INTERNAL_DB, false, tables, msg));
```

Comment:
keep the "empty" name case. In our current setting, it will be get two system tables.